### DISCUSSION TITLE
Rewrite in rust?

### DISCUSSION THREAD

**Discussion by nem0** [rokups/rbfx#247] [Ideas]
Have you thought about rewriting this in rust? It's a great langauge and it would make the project safer to use

**Comment by eugeneko** [comment]
Does rust support archetypes?

**Reply by nem0** [reply]
Archetypes are so 2019

**Reply by rokups** [reply]
Whats 2020 thing? Does rust support corona?

**Reply by 1vanK** [reply]
xD
